pkg/ip/ip.go

```diff
@@ -750,6 +753,57 @@ func init() {
 	}
 }
 
+var excludedIPs []net.IP
+
+func initExcludedIPs() {
+	// We exclude below bad device prefixes from address selection ...
+	prefixes := []string{
+		"docker",
+	}
+	links, err := netlink.LinkList()
+	if err != nil {
+		return
```

Comment:
Maybe makes sense to log an error here?

Reply:
Probably true, but then I guess we should log also the many other places for similar calls which we don't do right now. :/